src/types/Error.ts

```diff
@@ -0,0 +1,5 @@
+export type Error = {
+  message: string,
+  type: string;
+  isDanger?: boolean,
+};
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
![image](https://user-images.githubusercontent.com/96409563/209864220-273bd17f-7aa3-4217-90f8-2f5429686cc4.png)